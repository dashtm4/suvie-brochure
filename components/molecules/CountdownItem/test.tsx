import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { CountdownItem } from '.'

describe('<CountdownItem />', () => {
  const children = 'Test children'

  it('should renders correct <CountdownItem />', () => {
    const wrapper = shallow(<CountdownItem>{children}</CountdownItem>)

    expect(wrapper.contains(children)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
