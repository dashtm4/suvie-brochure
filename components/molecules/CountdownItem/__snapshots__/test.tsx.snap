// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CountdownItem /> should renders correct <CountdownItem /> 1`] = `
<div
  className="w-8"
>
  Test children
</div>
`;
